Soft Econ Adjustments/common/defines/zzz_MoG_SEA_def.txt

```diff
@@ -80,7 +81,7 @@
 	BUILDING_PROFIT_TARGET_TO_WITHDRAW_CASH = 0.2 # 0.15			# If profit margin is this or below, buildings should withdraw money from the cash reserves to make up the difference
 		# Previously: BUILDING_CASH_RESERVES_WITHDRAW_PROFIT_TARGET before directional revamp in 1.8
 	OWNERSHIP_BUILDING_PROFIT_TARGET_TO_RAISE_WAGES = 0.5			# Overrides BUILDING_PROFIT_TARGET_TO_RAISE_WAGES for buildings that own other buildings
-	OWNERSHIP_BUILDING_PROFIT_TARGET_TO_LOWER_WAGES = 0.3			# Overrides BUILDING_PROFIT_TARGET_TO_LOWER_WAGES for buildings that own other buildings
+	OWNERSHIP_BUILDING_PROFIT_TARGET_TO_LOWER_WAGES = 0.2999			# Overrides BUILDING_PROFIT_TARGET_TO_LOWER_WAGES for buildings that own other buildings
```

Comment:
I would actually make this lower rather than change BUILDING_PROFIT_TARGET_TO_HIRE_EMPLOYEES